src/web-app/http.go

```diff
@@ -22,24 +23,47 @@ const (
 	indexPage   = "index.html"
 )
 
-// LaunchWebInterface begins listening on http://$host, for enabling remote web access
+// Begins listening on http://$host, for enabling remote web access
 // Does NOT use HTTPS
-func LaunchWebInterface(host, staticDir string, rt *httprouter.Router) error {
+func LaunchWebInterface(host, staticDir string, mux *http.ServeMux) error {
 	logger.Info("Starting web interface on http://%s", host)
 	logger.Warning("HTTPS not in use!")
-	// logger.Info("Web resources directory: %s", staticDir)
+	logger.Info("Web resources directory: %s", staticDir)
 
 	appLoc, err := determineResourcePath(staticDir)
 	if err != nil {
 		return err
 	}
 	logger.Debug("static dir:%s", appLoc)
 
-	rt.NotFound = http.FileServer(http.Dir(appLoc))
+	listener, err := net.Listen("tcp", host)
```

Comment:
Did you rewrite the code in http.go, this code will not compatible with api client.